task/06-conditions-n-loops-tasks.js

```diff
@@ -119,7 +134,9 @@ function isTriangle(a,b,c) {
  *  
  */
 function doRectanglesOverlap(rect1, rect2) {
-    throw new Error('Not implemented');
+    if (rect1.top < rect2.top + rect2.width && rect2.top < rect1.top + rect1.width &&
```

Comment:
just return boolean expression. Already mentioned this previously . It's only in this task. Find other place yourself and fix them